include/seqan3/alignment/pairwise/alignment_algorithm.hpp

```diff
@@ -134,7 +134,7 @@ class alignment_algorithm :
         assert(cfg_ptr != nullptr);
 
         // We need to allocate the score_matrix and maybe the trace_matrix.
-        this->allocate_score_matrix(first_range, second_range);
+        this->allocate_matrix(first_range, second_range);
```

Comment:
so both matrices (if present) are allocated here? Maybe we should call it `allocate_matrices`

Reply:
Well, just because the internal layout uses two vectors it does not mean that the interface just allocates the dynamic programming matrix, what ever that is in detail. And in fact for the score only computation only one matrix is allocated which is even just one column. So I would like to keep the name as is.